elliott/elliottlib/cli/create_cli.py

```diff
@@ -24,6 +24,9 @@
               type=click.Choice(elliottlib.constants.errata_valid_impetus),
               default='standard',
               help="Impetus for the advisory creation. 'standard' by default")
+@click.option("--sort",
```

Comment:
`--sort` lets me think more of ordering a list. 
Maybe `--advisory-key` is better? In any case, think all values in https://github.com/openshift-eng/ocp-build-data-validator/blob/master/validator/json_schemas/assembly_group_config.schema.json#L30 should be possible.

Reply:
if you are suggesting the naming, I can change it to `--advisory-property`, I don't remember I saw a type with `silentops` or `advance` before and they are not apply the errata template